pkg/constants/constants.go

```diff
@@ -201,7 +201,6 @@ var ResourcesToBlock = map[string]bool{
 	"haproxyloadbalancerconfigs.netoperator.vmware.com":       true,
 	"httproutes.networking.x-k8s.io":                          true,
 	"imagedisks.imagecontroller.vmware.com":                   true,
-	"images.imagecontroller.vmware.com":                       true,
```

Comment:
We should keep this line but make it a comment.  Also add another comment to say "Don't add this to the skip list" to avoid us from making the same mistake again.
// "images.imagecontroller.vmware.com":                       true,

Reply:
Done